### ISSUE TITLE
Please camelCase property names.

### ISSUE THREAD

**Issue by cowboy** [endpoints/ember-data-endpoints#2]
Right now, I need to use underscores in my property name, like `applicant_state`, for ember-data-endpoints to use the underlying `applicant_state_id` property:

``` js
DS.Model.extend({
  applicant_state: DS.belongsTo('applicant-state'),
});
```

But I'd like to be able to camelCase the property name, like `applicantState`, and have it still work:

``` js
DS.Model.extend({
  applicantState: DS.belongsTo('applicant-state'),
});
```

**Comment by tkellen** [comment]
I'm pretty strongly opposed to camelCasing the responses in the API, but that shouldn't be an issue to support this, right @bmac?

**Comment by bobholt** [comment]
Fixed via #4
